features/user/register.feature

```diff
@@ -11,7 +11,7 @@ Feature: Register
     And I fill in "Password" with "new_user"
     And I fill in "Repeat password" with "new_user"
     And I press "Register"
-    Then I should see "The user has been created successfully."
+    Then I should see "Hello new_user1!"
```

Comment:
That's not enough. You should test against email dispatching process as shown here: http://docs.behat.org/en/v2.5/cookbook/using_the_profiler_with_minkbundle.html